mountpoint-s3/src/main.rs

```diff
@@ -218,7 +218,7 @@ struct CliArgs {
 
     #[clap(
         long,
-        help = "Disable all logging",
+        help = "Disable all logging excluding standard output. If no output is desired, consider redirecting it.",
```

Comment:
Could this be confusing when using `--foreground` (which logs to stdout)?

Reply:
Yes, it could. This is because the behavior of this argument is different depending on whether it is running in the foreground or background.

What do you think of this:

> Disable all logging if running in the foreground. When running in the background, this argument do not prevent messages sent to stdout from being displayed. If no output is desired, consider redirecting it.

I'll have to update the MD file too.